app/src/main/java/com/example/flexioffice/geofencing/GeofencingService.kt

```
@@ -216,83 +222,83 @@ class GeofencingService : Service() {
+                            Log.e(TAG, "Error re-registering geofence", error)
+                            // Set status to inactive on errors
                             geofencingManager.removeGeofences()
                         },
                     )
                 } else {
-                    Log.d(TAG, "User hat keine Home-Location mehr - deaktiviere Geofencing")
+                    Log.d(TAG, "User has no home location anymore - disabling geofencing")
                     geofencingManager.removeGeofences()
                 }
             }
         } catch (e: TimeoutCancellationException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)